pkgs/applications/display-managers/slim/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, fetchpatch, cmake, pkgconfig, xorg, libjpeg, libpng
+, fontconfig, freetype, pam, dbus, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "slim-ng";
+  version = "1.3.7";
+
+  src = fetchFromGitHub {
+    owner = "oxij";
+    repo = pname;
+    rev = "b6d77d41eeb796d3788ecab2d02e616c858661e1";
+    sha256 = "1rvrppc7n57d18022v7hmv3vglhsyb1nkql5qw6ka48wrrb2vcj7";
+  };
+
+  patches = [
+    # Allow the paths of the configuration file and theme directory to
+    # be set at runtime.
+    ./runtime-paths.patch
+  ];
+
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace /lib $out/lib
+  '';
+
+  cmakeFlags = [ "-DUSE_PAM=1" ];
+
+  enableParallelBuilding = true;
+
+  buildInputs =
+    [ cmake pkgconfig libjpeg libpng fontconfig freetype
```

Comment:
Build platform dependencies like `cmake` and `pkg-config` should go to `nativeBuildInputs`